### ISSUE TITLE
Microsoft.VSCode.CPP.Extension.linux 0.10.5 SIGABRT on Fedora 25 x64

### ISSUE THREAD

**Issue by pquantin** [microsoft/vscode-cpptools#611]
Hi,
since today C++ extension has become unusable for me: it keeps crashing on my project (but seems to run find on another project). In case it matters, I'm running Code 1.11.0. I'm attaching the corresponding ABRT folder and here is the call stack I get when loading the coredump in gdb:

Program terminated with signal SIGABRT, Aborted.
```
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:58
58	}
[Current thread is 1 (Thread 0x7fbc4e7fc700 (LWP 17952))]
(gdb) bt
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:58
#1  0x00007fbc60c2c51a in __GI_abort () at abort.c:89
#2  0x0000000000adeac7 in abort_message ()
#3  0x0000000000adebcd in default_terminate_handler() ()
#4  0x0000000000aa3253 in std::__terminate(void (*)()) ()
#5  0x0000000000aa2a1c in __cxa_throw ()
#6  0x00000000005569c7 in std::__1::__basic_string_common<true>::__throw_out_of_range() const ()
#7  0x00000000006e414e in symbol_search_engine::search_results_t::impl::get_next_result(code_store::a_record_id<code_store::an_item_tag>&, code_store::a_record_id<code_store::a_file_tag>&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >*, code_store::a_code_item_kind&, microsoft::vc::guid_t&, int&, symbol_search_engine::match_kind_e&, bool&, symbol_search_engine::result_status_e&) ()
#8  0x00000000005972ab in browse_engine::query_symbols(char const*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const*, std::__1::vector<browse_symbol, std::__1::allocator<browse_symbol> >&) ()
#9  0x0000000000574bc8 in message_handler::textDocument_documentSymbol(vscode::DidOpenTextDocumentParams) ()
#10 0x00000000005744c3 in vscode::handler_base<message_handler>::handle_symbol(vscode::vscode_client_message&&) ()
#11 0x0000000000574030 in vscode::handler_base<message_handler>::main_loop()::{lambda()#4}::operator()() const ()
#12 0x0000000000573f9d in void* std::__1::__thread_proxy<std::__1::tuple<vscode::handler_base<message_handler>::main_loop()::{lambda()#4}> >(std::__1::tuple<vscode::handler_base<message_handler>::main_loop()::{lambda()#4}>) ()
#13 0x00007fbc60fc26ca in start_thread (arg=0x7fbc4e7fc700) at pthread_create.c:333
#14 0x00007fbc60cfcf7f in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:105```

[ccpp-2017-04-06-15:13:37-17940.zip](https://github.com/Microsoft/vscode-cpptools/files/902658/ccpp-2017-04-06-15.13.37-17940.zip)

**Comment by sean-mcmanus** [comment]
Thanks a lot for reporting this issue. The crash repros when the "_" identifier is used and symbol search is done. We'll fix it in a future update.

**Comment by bobbrow** [comment]
This issue should be fixed with v0.11.0.  Please let us know if it is not.
